### ISSUE TITLE
Decide what color navigation bar should be in private mode

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#438]
We are currently using a the top gradient color for the background of the toolbar. Because of this, our navigation bar must be the same color for it to blend with the toolbar in the browser view. Unfortunately this means it **does not** blend nicely on the home view. 

## Previously
<img width=300 src=https://user-images.githubusercontent.com/4400286/52588576-aaf40d80-2df1-11e9-8d63-de5a2d4d0bd7.png>

## Currently
<img width=300 src=https://user-images.githubusercontent.com/4400286/52588289-f4902880-2df0-11e9-9486-65cf6afaf435.png>
<img width=300 src=https://user-images.githubusercontent.com/4400286/52588291-f528bf00-2df0-11e9-8fed-4af1402fb6b5.png>

The original mocks had a white navigation bar, but that seemed to clash with the rest of the darker color scheme. 

@brampitoyo any thoughts on this? 😄

**Comment by brampitoyo** [comment]
@sblatz Apologies for not specifying the navigation bar colour, and thanks for not forgetting this little but important design detail!

For guidance, let’s look at how things are done in normal browsing.

The current normal home screen shows a navigation bar that’s of a different colour than the background gradient.

<img width="240" src="https://user-images.githubusercontent.com/51007/52601850-2e326380-2ec5-11e9-9063-64991596dd23.png">

This `grey-10` navigation bar blends with the `grey-10` toolbar when you’re browsing.

<img width="240" src="https://user-images.githubusercontent.com/51007/52601890-58842100-2ec5-11e9-9abf-f8cdda5831b6.png">

Let’s apply the same principle to private browsing.

When browsing privately, we should (as you wrote) “[use] the top gradient color for the background of the toolbar”.

<img width="240" src="https://user-images.githubusercontent.com/51007/52601995-b6186d80-2ec5-11e9-869a-d6281bfabd93.png">

Of course, as a result, “[the navigation bar] **does not** blend nicely on the home view”. But it will blend when you’re browsing sites, as shown in your example:

<img width="240" src="https://user-images.githubusercontent.com/4400286/52588289-f4902880-2df0-11e9-9486-65cf6afaf435.png">

In summary, the colour choice you’ve made was the right one!

**Comment by sblatz** [comment]
Thanks for your input! Closing as resolved

**Comment by sblatz** [comment]
Per a conversation with @brampitoyo, it sounds like we want to do a dark bar in the browser fragment and the blended gradient on the home view.
